webapp/src/main/resources/public/js/components/jobs/JobsView.jsx

```diff
@@ -84,7 +93,8 @@ class JobsView extends React.Component {
                 <div className="ptm">
                     {
                         this.state.jobs
-                            .filter(job => this.state.filter.length === 0 ? true : this.state.filter.includes(job.repository))
+                            .filter(job => (this.state.filter.length === 0 ? true : this.state.filter.includes(job.repository))
+                                && this.state.jobType === job.type)
```

Comment:
If the jobType was not selected, this would always be false and no values would be shown?